src/System/Components/PageLoadingBar.tsx

```diff
@@ -44,18 +47,19 @@ export const PageLoadingBar: React.FC<PageLoadingBarProps> = ({
         ref={barRef as any}
         loading={loading}
         isComplete={isComplete}
+        scaleX={scaleX}
         backgroundColor="brand"
         height={2}
         top="1px"
-        left={0}
-        width={isComplete ? barRef.current?.clientWidth : 0}
-        zIndex={100}
+        left="0px"
+        zIndex={1000}
```

Comment:
https://github.com/artsy/force/blob/main/src/Apps/Components/constants.ts

Reply:
Ah yah, actually this isn't needed at all and is irrelevant. I used Z.pageLoadBar up above 👍 Will remove

Reply:
https://github.com/artsy/force/pull/14224